hs_core/templates/pages/genericresource.html

```diff
@@ -25,25 +25,31 @@
 
         {# ======= Missing fields notification =======#}
         {% if not metadata_form and page.perms.change %}
-            {% if missing_metadata_elements or title|stringformat:"s" == "Untitled resource" %}
+            {% if missing_metadata_elements or title|stringformat:"s" == "Untitled resource" or not cm.has_required_content_files %}
                 <div class="col-sm-12">
                     <div class="alert {% if just_created %}alert-success{% else %}alert-warning{% endif %} alert-dismissible" role="alert">
                         <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span
                                 aria-hidden="true">&times;</span></button>
-                        {% if just_created %}
-                            <strong>Congratulations! </strong>
-                            <span>Now that your resource has been created the following metadata are still required to make it public or discoverable:</span>
-                        {% else %}
-                            <span>The following metadata are still required to make the resource public or discoverable:</span>
-                        {% endif %}
-                        <ul>
-                            {% for element in missing_metadata_elements %}
-                                <li>{{ element }}</li>
-                            {% endfor %}
-                            {% if title|stringformat:"s" == "Untitled resource" %}
-                               <li>Title: needs to be changed</li>
+                        {% if missing_metadata_elements  %}
+                            {% if just_created %}
+                                <strong>Congratulations! </strong>
+                                <span>Now that your resource has been created the following metadata are still required to make it public or discoverable:</span>
+                            {% else %}
+                                <span>The following metadata are still required to make the resource public or discoverable:</span>
                             {% endif %}
-                        </ul>
+                            <ul>
+                                {% for element in missing_metadata_elements %}
+                                    <li>{{ element }}</li>
+                                {% endfor %}
+                                {% if title|stringformat:"s" == "Untitled resource" %}
+                                   <li>Title: needs to be changed</li>
+                                {% endif %}
+                            </ul>
+                        {%  endif %}
+                        {%  if not cm.has_required_content_files %}
+                                <br>
+                                <span>Content file(s) is missing. Content file(s) is required to make the resource public or discoverable.</span>
```

Comment:
I think "Content file(s) is missing" can be removed from the message to keep consistency with metadata message above. Should we just say "Content files are required..." without parenthesis (s)?